pkgs/applications/misc/smb3-foundry/fix-relative-dirs.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/foundry/gui/WarningList.py b/foundry/gui/WarningList.py
```

Comment:
Can this patch be upstreamed?

Reply:
It could be, I'll open a PR

Reply:
Okay, I see that the repo is quite active, so I'll wait out the PR being merged, and then use fetchpatch